force_wfmanager/wfmanager.py

```diff
@@ -47,8 +61,15 @@ def close(self):
         fails, the application is not closed so he has a chance to try to
         save again. Overrides close from pyface.tasks.task_window """
 
-        # The attached wfmanager_task for saving methods
-        wfmanager_task = self.tasks[0]
+        # The attached wfmanager_setup_task for saving methods
```

Comment:
This will not be needed when we have a BaseTask class for the common actions of the Setup & Result tasks. But that's for another PR maybe!